test/screenshot/top-app-bar/prominentToShortCollapsed.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import TopAppBar from '../../../packages/top-app-bar';
+import MaterialIcon from '../../../packages/material-icon';
+import MainTopAppBarContent from './mainContent';
+
+class TopAppBarProminentToShortCollapsedScreenshotTest extends React.Component {
+  state = {
+    isPhone: window.innerWidth < 599,
+  };
+
+  componentDidMount() {
+    window.addEventListener('resize', this.updateTopAppBarVariant);
+  }
+
+  shouldComponentUpdate(_, nextState) {
+    if (nextState.isPhone === this.state.isPhone) {
```

Comment:
can you just 
```
return nextState.isPhone !== this.state.isPhone
```


If you think its too confusing don't worry about it.

Reply:
Not confusing at all. I lifted this test from [drawer's permanentToModal](https://github.com/material-components/material-components-web-react/blob/2ee4c80efb514f86fbab0504e183461b6ee5605d/test/screenshot/drawer/permanentToModal.js#L14-L19) test. That should probably be changed as well.